### PR TITLE
solo game initial scoring system

### PR DESCRIPTION
**Pull request by gouws-j**
the scoring algorithm has been added. Still working the bootstrap to show the score. (Can Bootstrap and databases even?)

relating to #20 and #19. I think it relates to both.
[pull_request_checklist_20.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8797712/pull_request_checklist_20.md)

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -113,15 +119,17 @@ document.addEventListener('DOMContentLoaded', () => {
...
       if (numGuesses.length === 6) {
-        window.alert(`Better Luck Next Time! The word is ${word}.`)
+        answer = false
+        setTimeout(() => {window.alert(`Better Luck Next Time! The word is ${word}. Your score is: ${calcScore()}`);} , 1000) 
       }
       numGuesses.push([])
 
     }).catch(() => {
-      window.alert("word is not recognised")
+      window.alert("Word is doesn't exist. Try again!")
```
Comment (robinjonker) [inline public/main.js:132]: Please fix the typo in the string

File: public/styles.css [hunk 2]
```diff
@@ -144,4 +144,5 @@ header {
   align-items: left;
   flex-grow: 1;
   overflow: hidden;
-}
\ No newline at end of file
+}
```
Comment (robinjonker) [inline public/styles.css:147]: Remove this change from the merge as it is not changing anything

File: sessions/6Grp37Q0gXyUthPqdC9gM-AZNJUZk4z9.json [hunk 3]
```diff
@@ -1 +0,0 @@
-{"cookie":{"originalMaxAge":null,"expires":null,"httpOnly":true,"path":"/"},"ID":1,"__lastAccess":1653573486605}
```
Comment (robinjonker) [inline sessions/6Grp37Q0gXyUthPqdC9gM-AZNJUZk4z9.json:1]: Can both sessions files be excluded from the merge please

File: views/users/solo_game.ejs [hunk 4]
```diff
@@ -91,37 +90,46 @@
...
+                    </svg>
+                </button>
+                <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton1">
+                    <li><a class="dropdown-item" href="/home" type="sologame" id="sologame" name="sologame">Go to Home
+                            Page</a></li>
+                    <li><a class="dropdown-item" href="/leaderboard" type="leaderboard" id="leaderboard"
+                            name="leaderboard">Leaderboard</a></li>
+                    <li><a class="dropdown-item" href="/settings" type="settings" id="settings"
+                            name="settings">Settings</a></li>
```
Comment (robinjonker) [inline views/users/solo_game.ejs:129]: Why were these changes made in the dropdown button of the solo game?

**Review by robinjonker — CHANGES_REQUESTED** [review]
Functionality added for scoring looks good, just small formating changes are required

File: views/users/solo_game.ejs (hunk 4 above)
Comment (gouws-j) [reply views/users/solo_game.ejs:129]: I think I got confused when merging from main. I quickly look at the code in main and compare.

File: public/styles.css (hunk 2 above)
Comment (gouws-j) [reply public/styles.css:147]: Will do.

**Review by robinjonker — APPROVED** [review]
[Code_Review_102.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8798024/Code_Review_102.md)

**Pull request merged**